Formula/node_exporter.rb

```diff
@@ -14,13 +14,30 @@ class NodeExporter < Formula
 
   depends_on "go" => :build
 
+  resource "promu" do
+    # check and update upon next release
+    # (matches <https://github.com/prometheus/node_exporter/blob/v0.17.0/Makefile.common#L68-L69>)
+    url "https://github.com/prometheus/promu/releases/download/v0.2.0/promu-0.2.0.darwin-amd64.tar.gz"
+    sha256 "6372caea7d8e7c0f208f6c040e2bfb462c72c9f7b2f6c13b7992d7caa5fbcf75"
+  end
+
+  # Fix checksum mismatch of `soundcloud/go-runit@v0.0.0-20150630195641-06ad41a06c4a`.
+  patch do
+    url "https://github.com/prometheus/node_exporter/commit/97dab59e18c575d1384a9afd54e9f30473f34322.patch?full_index=1"
```

Comment:
Why is that necessary? Please confirm the issue with upstream, to make sure it's also fixed on their end.

Reply:
[According to the patch author](https://github.com/prometheus/node_exporter/issues/1199):
> the golang checksum code changed between 1.11.3 and 1.11.4 to correctly handle symlinks, and thus broke `go.sum` for a bunch of projects.

And upstream is definitely aware—the patch is from a commit already present in the project’s `master` branch. (And so the patch can be removed with the next release of `node_exporter`.)

Is that sufficient? Would you prefer for the explanatory comment to have more detail?

Reply:
OK, please add link to the issue in comment